src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PersonalDetailServiceTest.kt

```diff
@@ -61,74 +62,88 @@ class PersonalDetailServiceTest {
     runBlockingTest {
       val crn = "my wonderful crn"
-
       given(communityApiClient.getAllOffenderDetails(anyString()))
-        .willReturn(Mono.fromCallable { allOffenderDetailsResponse() })
+        .willReturn(Mono.fromCallable { allOffenderDetailsResponse })
       given(communityApiClient.getRegistrations(anyString()))
         .willReturn(Mono.empty())
 
-      personDetailsService.getPersonDetails(crn)
-      // TODO assert details
+      val response = personDetailsService.getPersonDetails(crn)
+
+      val personalDetails = response.personDetails!!
+      val currentAddress = response.currentAddress!!
+      val riskFlagsShouldBeEmpty = response.risk!!.flags
+      val offenderManager = response.offenderManager!!
 
+      assertThat(personalDetails.crn).isEqualTo(crn)
+      assertThat(personalDetails.age).isEqualTo(39)
+      assertThat(personalDetails.gender).isEqualTo("Male")
+      assertThat(personalDetails.dateOfBirth).isEqualTo(LocalDate.parse("1982-10-24"))
+      assertThat(personalDetails.name).isEqualTo("John Smith")
+      assertThat(currentAddress.line1).isEqualTo("32 HMPPS Digital Studio")
+      assertThat(currentAddress.line2).isEqualTo("Sheffield City Centre")
+      assertThat(currentAddress.town).isEqualTo("Sheffield")
+      assertThat(currentAddress.postcode).isEqualTo("S3 7BS")
+      assertThat(offenderManager.name).isEqualTo("Sheila Linda Hancock")
+      assertThat(offenderManager.email).isEqualTo("first.last@digital.justice.gov.uk")
+      assertThat(offenderManager.phoneNumber).isEqualTo("09056714321")
+      assertThat(offenderManager.probationTeam?.code).isEqualTo("C01T04")
+      assertThat(offenderManager.probationTeam?.label).isEqualTo("OMU A")
+      assertThat(riskFlagsShouldBeEmpty).isNull()
       then(communityApiClient).should().getAllOffenderDetails(crn)
     }
   }
 
-  // TODO should throw exception with registration only
-
-  private fun allOffenderDetailsResponse(): AllOffenderDetailsResponse {
-    return AllOffenderDetailsResponse(
-      dateOfBirth = LocalDate.parse("1982-10-24"),
-      firstName = "John",
-      surname = "Smith",
-      gender = "Male",
-      contactDetails = ContactDetails(
-        addresses = listOf(
-          Address(
-            postcode = "S3 7BS",
-            district = "Sheffield City Centre",
-            addressNumber = "32",
-            buildingName = "HMPPS Digital Studio",
-            town = "Sheffield",
-            county = "South Yorkshire", status = AddressStatus(code = "ABC123", description = "Main")
-          ),
-          Address(
-            town = "Sheffield",
-            county = "South Yorkshire",
-            buildingName = "HMPPS Digital Studio",
-            district = "Sheffield City Centre",
-            status = AddressStatus(code = "ABC123", description = "Not Main"),
-            postcode = "S3 7BS",
-            addressNumber = "33"
-          )
+  private val allOffenderDetailsResponse = AllOffenderDetailsResponse(
```

Comment:
Probably need to add a happy path test for when 'AllOffenderDetails' and 'Registrations' are both retrieved.

Reply:
Done